javascript/javascript3/week3/shoppingCart.js

```diff
@@ -0,0 +1,107 @@
+class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
+    convertToCurrency(currency){
+      let price=0;
+      if(currency==="dollars"){
+        price=this.price/6.66;
+      }
+      else if(currency==="rial"){
+        price=this.price*46800;
+      }
+      else if(currency==="euro"){
+        price=this.price/7.5;
+      }else{
+        price=this.price;
+      }
+      return  price;
+    }
+  }
+  
+  class ShoppingCart {
+    constructor() {
+      this.products = [];
+    }
+  
+    addProduct(product) {
+      // Implement functionality here
+      this.products.push(product);
+    }
+  
+    removeProduct(product) {
+      this.products=this.products.filter(function(p){return (p.name!==product.name);});
```

Comment:
can be cleaner rewritten as an arrow function. 
...filter(p => p.name!== product.name)